tests/page-objects/forms/generic-form.po.js

```diff
@@ -109,20 +124,23 @@ module.exports = {
     const submitButton = element(by.css('.btn.submit.btn-primary'));
     await helper.waitElementToBeClickable(submitButton);
     await submitButton.click();
-    await helper.waitElementToBeVisible(element(by.css('div#reports-content')));
+    await helper.waitElementToPresentNative(element(by.css('div#reports-content')));
     const details = element(by.css('div.details'));
     await helper.waitUntilReadyNative(details);
     expect(await details.isPresent()).toBeTruthy();
   },
 
-  validateReport: () => {
-    const reportValidBtn = element(by.css('[ng-include*="verify-valid"]'));
-    helper.waitElementToBeClickable(reportValidBtn);
-    reportValidBtn.click();
+  validateReportNative: async () => {
+    const reviewButton = element.all(by.css('.mm-icon-inverse')).get(0);
```

Comment:
Does this still need to be here? isn't the review interface already open from `reportApproveNative`'s call?

Reply:
No. Forgot to clean up. Good catch.